### PR TITLE
Fix Python 3 dict object to list, described in issue #5488

### PR DESCRIPTION
**Pull request by zuoxingdong**
It is also compatible with Python 2, converting a list to a list is a list.

### CODE REVIEW

**Comment by sguada** [comment]
@nathansilberman can you take a look at this change?
Is it expected that eval_ops are always a list/tuple, or can it be a dict()?

File: tensorflow/contrib/slim/python/slim/evaluation.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def evaluate_once(master,
       scaffold=monitored_session.Scaffold(
           init_op=initial_op,
           init_feed_dict=initial_op_feed_dict),
-      eval_ops=eval_op,
+      eval_ops=list(eval_op),
```
Comment (sguada) [inline tensorflow/contrib/slim/python/slim/evaluation.py]: eval_op could be a Tensor, a List or a Dict, isn't it?

**Comment by drpngx** [comment]
@nathansilberman do you have cycles to take a look?

**Comment by drpngx** [comment]
@zuoxingdong in the meanwhile, please resolve conflicts and push again.

**Comment by zuoxingdong** [comment]
@drpngx There are 2 conflicts with saver=saver, now they are resolved.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by vrv** [comment]
It looks like there are still failures -- can you fix the failures and validate that the relevant tests are passing?

**Pull request closed without merging**